### PR TITLE
Update driver version to 0.1.0-alpha

### PR DESCRIPTION
**Pull request by leakingtapan**
Fixes: #82

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,12 +13,12 @@
...
 
 IMAGE=amazon/aws-ebs-csi-driver
-VERSION=testing
+VERSION="0.1.0-alpha"
 
 .PHONY: aws-ebs-csi-driver
 aws-ebs-csi-driver:
 	mkdir -p bin
-	go build -o bin/aws-ebs-csi-driver ./cmd/aws-ebs-csi-driver
+	go build -ldflags "-X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.VendorVersion=${VERSION}" -o bin/aws-ebs-csi-driver ./cmd/
```
Comment (bertinatto) [inline Makefile]: This won't the binary binary that goes into the image. The `Dockerfile` is not calling `make`, it's calling `go build` directly.

We should change the `Dockerfile` to use `make` instead. We need to install it first, if we want to keep using `golang:1.11.1-alpine3.8`.

File: Makefile [hunk 2]
```diff
@@ -13,12 +13,12 @@
 # limitations under the License.
 
 IMAGE=amazon/aws-ebs-csi-driver
-VERSION=testing
+VERSION="0.1.0-alpha"
```
[the remark this answers is not in the document]
Comment (leakingtapan) [reply Makefile]: done. Out of some reason, I got nervous when not adding quote 🤣

File: pkg/driver/driver.go [hunk 3]
```diff
@@ -29,9 +29,14 @@ import (
...
-	vendorVersion = "0.0.1" // FIXME
-	topologyKey   = driverName + "/zone"
+	driverName  = "com.amazon.aws.csi.ebs"
+	topologyKey = driverName + "/zone"
+)
+
+var (
+	// verion of the driver
+	// value is set during build
+	VendorVersion string
```
[the remark this answers is not in the document]
Comment (leakingtapan) [reply pkg/driver/driver.go]: done

File: Makefile (hunk 1 above)
Comment (leakingtapan) [reply Makefile]: updated to use golang1.11.1-stretch as build image

**Pull request merged**